ts/src/binance.ts

```diff
@@ -4815,6 +4815,9 @@ export default class binance extends Exchange {
             //     }
             //   ]
         }
+        for (let i = 0; i < response.length; i++) {
+            response[i]['type'] = 'deposit';
```

Comment:
by doing this we're modifying the original response which we avoid at all costs,  is this really necessary?  (sorry if I misled you initially I was not suggesting it)

Reply:
I agree it's not the best, but it's the only way I saw to pass a parameter to `parseTransactions` as we don't have params, and to let it know if it's parsing deposits or transactions. What solution would you recommend if not?